packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -556,7 +558,7 @@ export class AccountResolver {
         } else {
           results.push({
             taskName,
-            status: 'Could not locate task.',
+            status: 'Task completed and no longer in queue.',
```

Comment:
I don't know whole context here but presumably we know that the task did exist because of task name?

Reply:
That's correct. There's a preceding call that returns the set of tasks kicked off, so _presumably_ it existed at some point.